adapters/play/src/test/scala/caliban/PlayAdapterSpec.scala

```diff
@@ -24,7 +24,7 @@ import zio.test.{ Live, ZIOSpecDefault }
 import scala.language.postfixOps
 
 object PlayAdapterSpec extends ZIOSpecDefault {
-  import sttp.tapir.json.play._
```

Comment:
Play can't use play? 😭

Reply:
It was already an issue - it just became obvious now that `tapir-json-play` updated play to `2.10.1`. We didn't realise earlier cause we don't have `play-json` as a dependency in the tests

I think we're fine since our compile dependencies are okay

Reply:
Play 2.9.0 seems to be nearing (with Scala 3 support!) so just need to wait a bit more i guess. It's currently at RC2, although if zio-http is any indication it might take a fair bit till it makes it to a stable version